src/views/News/components/NewsList/index.jsx

```diff
@@ -66,42 +40,75 @@ const NewsList = ({ news, personalUnapprovedNews, handleNewsItemEdit, handleNews
     };
   });
 
-  return news.length > 0 || personalUnapprovedNews.length > 0 ? (
-    <Card>
-      {width < BREAKPOINT_WIDTH ? singleHeader : fullHeader}
-      <Grid>
-        <List className={styles.news_list} disablePadding>
-          {personalUnapprovedNews.length > 0 &&
-            personalUnapprovedNews.map((posting) => (
-              <NewsItem
-                posting={posting}
-                unapproved
-                size={width < BREAKPOINT_WIDTH ? 'single' : 'full'}
-                handleNewsItemEdit={handleNewsItemEdit}
-                handleNewsItemDelete={handleNewsItemDelete}
-                key={posting.SNID}
-              />
-            ))}
+  //https://www.pluralsight.com/guides/re-render-react-component-on-window-resize
+  //Excellent resource for handling rerender on resize -Josh
+
+  const singleHeader = (
+    <div className={styles.header}>
+      <Grid container direction="row">
+        <Grid item xs={12}>
+          <Typography variant="body2" className={styles.header}>
+            {header}
+          </Typography>
+        </Grid>
+      </Grid>
+    </div>
+  );
 
-          {news.length > 0 &&
-            news.map((posting) => (
-              <NewsItem
-                posting={posting}
-                //approved
-                size={width < BREAKPOINT_WIDTH ? 'single' : 'full'}
-                handleNewsItemEdit={handleNewsItemEdit}
-                handleNewsItemDelete={handleNewsItemDelete}
-                key={posting.SNID}
-              />
-            ))}
-        </List>
+  const fullHeader = (
+    <Grid container direction="row" className={styles.header}>
+      <Grid item xs={2}>
+        <Typography variant="body2" className={styles.header}>
+          CATEGORY
+        </Typography>
+      </Grid>
+      <Grid item xs={5}>
+        <Typography variant="body2" className={styles.header}>
+          SUBJECT
+        </Typography>
+      </Grid>
+      <Grid item xs={3}>
+        <Typography variant="body2" className={styles.header}>
+          POSTED BY
+        </Typography>
+      </Grid>
+      <Grid item xs={2}>
+        <Typography variant="body2" className={styles.header}>
+          POSTED
+        </Typography>
       </Grid>
+    </Grid>
+  );
```

Comment:
Why did you move this inside of the component? If it is constant and not dependent on params/members of the component, it should be outside where it was.